api/index.ts

```diff
@@ -48,8 +54,32 @@ export async function login(user, password) {
   });
 }
 
-export function getRequestsHeaders() {
+export async function vote(user, score) {
+  const payload = {
+    hospitalId: user.hospitalId,
+    userId: user.id,
+    score,
+  };
+  const fetchOptions = {
+    method: "POST",
+    headers: getRequestsHeaders(user.token),
+    body: JSON.stringify(payload),
+  };
+
+  return fetch(ENDPOINTS.VOTE, fetchOptions).then(requestToJson);
+}
+
+export function getRequestsHeaders(token = null) {
   return {
     "Content-Type": "application/json",
+    Authorization: token ? `Bearer ${token}` : undefined,
   };
 }
+
+export async function requestToJson(res) {
```

Comment:
I added this function because I changed the way we send API errors to not lose the error message from the backend:
```diff
- "Not allowed"
+ { error: "Not allowed" }
```
We receive the error message "Error parsing the JSON" when we try to parse the response from the backend and we were losing the error message from it